packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -250,6 +250,15 @@ const ExpandableItemBase = kind({
 		 */
 		open: PropTypes.bool,
 
+		/**
+		 * Prevents rendering the transition container on mount.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @private
+		 */
+		preventTransitionContainerRender: PropTypes.bool,
```

Comment:
This prop name is a bit unconventional in the framework. It's very specific to the implementation details of the effect. What are some other options we should consider that are a bit more generalized? Maybe `hideChildren` as is used in Panel?